config/locales/da-DK.yml

```diff
@@ -21,7 +21,7 @@ da-DK:
       sended: "En email er blevet sendt til '%{email}' med instruktioner for at nulstille dit kodeord."
       user_not_found: "Kan ikke finde en bruger med '%{email}'."
       password_not_required: "Denne bruger kræver ikke et kodeord. Log ind med '%{provider}' konto i stedet."
-      missing_passwords: "Du skal fylde alle felter ud som indeholder 'Password' og 'Password confirmation'."
+      missing_passwords: "Du skal udfylde alle felter som indeholder 'Password' og 'Password confirmation'."
```

Comment:
Perhaps use "Kodeord" here as well, for consistency. Don't know what "Password confirmation" is translated to in danish.

Reply:
This is a harder problem, which a trickier wording could work around, so that we never mentioned the expected labels of those fields.